docs/api/scheduler.md

```diff
@@ -271,6 +271,26 @@ For the default queue hierarchy (only `root.default` leaf queue exists) a simila
 
 ## Applications
 
+### Partition applications
+
+Fetch all Applications for the given Partition/State combination and displays general information about the applications like used resources, queue name, submission time and allocations.
+Only following application states are allowed: active, rejected, completed. Active is a fake state that represents all application states except completed and rejected.
+For active state, can narrow the result by status query parameters.
+
+**URL** : `/ws/v1/partition/{partitionName}/applications/{state}?status`
```

Comment:
IIRC, the url format should be `/ws/v1/partition/:partition/applications/:state`, right?